JitsiConference.js

```diff
@@ -2530,6 +2553,57 @@ JitsiConference.prototype._onIceConnectionEstablished = function(
 
 };
 
+/**
+ * Called when the chat room reads a new list of properties from jicofo's
+ * presence. The properties may have changed, but they don't have to.
+ *
+ * @param {Object} properties - The properties keyed by the property name
+ * ('key').
+ * @private
+ */
+JitsiConference.prototype._propertiesChanged = function(properties) {
+    const changed = isEqual(properties, this.properties);
+
+    this.properties = properties;
+    if (changed) {
+        this.eventEmitter.emit(JitsiConferenceEvents.PROPERTIES_CHANGED);
+
+        // Some of the properties need to be added to analytics events.
+        const analyticsKeys = [
+
+            // The number of jitsi-videobridge instances currently used for the
+            // conference.
+            'bridge-count',
+
+            // The conference creation time (set by jicofo).
+            'created-ms',
+            'octo-enabled'
+        ];
+
+        analyticsKeys.forEach(key => {
+            if (properties[key] && properties[key].value !== undefined) {
+                Statistics.analytics.addPermanentProperties({
+                    [key.replace('-', '_')]: properties[key].value
+                });
+            }
+        });
+    }
+};
+
+/**
+ * Gets a conference property with a given key.
+ *
+ * @param {string} key - The key.
+ * @returns {*} The value
+ */
+JitsiConference.prototype.getProperty = function(key) {
+    if (!this.properties) {
```

Comment:
Is there not always a this.properties defined? You're adding this for safety?

Reply:
It's true that it is always defined with today's code, but it depends on code in another file. That is, if ChatRoom.js decides to emit an event with undefined, this.properties will be undefined. So I'd rather keep the check for safety.